data-shaping/output/output.py

```diff
@@ -642,26 +642,26 @@ def transfer_training_record_separate(path, id, hei_qualification_completed, hei
                 "hei_awarding_institution_id": hei_awards,
                 "hei_extension": hei_extension,
                 "hei_extension_date": hei_extension_date,
-                "hei_extension_date_comments": hei_extension_date_comments,
+                "hei_extension_comments": hei_extension_comments,
                 "program_certification": program_certification,
                 "program_certification_date": program_certification_date,
                 "program_leaving_date": program_leaving_date,
                 "program_leaving_reason": program_leaving_reason,
                 "program_leaving_comments": program_leaving_comments,
                 "hsst_pathway_id": hsst_pathway,
                 "hsst_portfolio_completed": hsst_portfolio_completed,
-                "hsst_portfolio_completion_date": hsst_portfolio_completion_date,
+                "hsst_portfolio_compl_date": hsst_portfolio_completion_date,
                 "hsst_expected_exit_date": hsst_expected_exit,
                 "hsst_arp_completed": hsst_arp_completed,
                 "hsst_arp_completion_date": hsst_arp_completion_date,
-                "hsst_d_clin_part_a_completed": hsst_d_clin_part_a_completed,
-                "hsst_d_clin_part_a_completion_date": hsst_d_clin_part_a_completion_date,
-                "hsst_d_clin_part_b_completed": hsst_d_clin_part_b_completed,
-                "hsst_d_clin_part_b_completion_date": hsst_d_clin_part_b_completion_date,
-                "hsst_d_clin_part_c1_completed": hsst_d_clin_part_c1_completed,
-                "hsst_d_clin_part_c1_completion_date": hsst_d_clin_part_c1_completion_date,
-                "hsst_d_clin_part_c2_completed": hsst_d_clin_part_c2_completed,
-                "hsst_d_clin_part_c2_completion_date": hsst_d_clin_part_c2_completion_date,
+                "hsst_dclin_part_a_completed": hsst_d_clin_part_a_completed,
```

Comment:
I'm happy to change these but we're mixing three variations of "completed", "comp" and "compl". Could we use just one?

Reply:
this is only due to length constrain...

if you prefer - I can change it all to `comp`?

Reply:
If we have to bend due to to the length constraint then `comp` is fine